javascript/ql/test/library-tests/CallGraphs/AnnotatedTest/method-on-object.js

```diff
@@ -0,0 +1,24 @@
+/** name:foo */
```

Comment:
Try putting `import 'dummy';` at the top so the file is seen as a module. Otherwise all variables at the top-level become global variables which are tracked less precisely.

Reply:
That changed initial test results (in positive way), but I still don't see impact of implicit receiver step in the test changes after merging in the PR: https://github.com/github/codeql/pull/15883/commits/1e7d72ef401191f984cefcb229c9a027e5d68ec4

Reply:
Previously there was a requirement that the host object was an object literal or function. That restriction was lifted in the PR, but in these test, it seems the host object is always an object literal, so they are not affected by the change.